src/visu_config.xsd

```diff
@@ -563,7 +564,12 @@
             <xsd:appinfo>level:expert</xsd:appinfo>
         </xsd:annotation>
     </xsd:attribute>
-    <xsd:attribute name="visible" type="xsd:boolean" />
+    <xsd:attribute name="visible" type="xsd:boolean">
+    	<xsd:annotation>
+        	<xsd:documentation xml:lang="en">Displays the widget (page name) which links to be page</xsd:documentation>
```

Comment:
I would suggest "Show a widget, which links to the page" I don't think that it is clear what the text in the brackets means, and I think we do not need it here.